### PR TITLE
copyright clarifications

### PR DESCRIPTION
**Pull request by arcuri82**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -267,7 +267,9 @@ e.g., details on the Many Independent Algorithm (MIO).
 
 ### License
 _EvoMaster_'s source code is released under the LGPL (v3) license.
-
+For a list of the used third-party libraries, you can directly see the root [pom.xml](./pom.xml) file.
+For a list of code directly imported (and then possibly modified/updated) from 
+other open-source projects, see [here](./docs/reused_code.md).
 
 
 ### ![](https://www.yourkit.com/images/yklogo.png)
```
Comment (jgaleotti) [inline README.md]: One of the libraries used by the regex distance code has BSD license, would that be compatible?

File: docs/for_developers.md [hunk 2]
```diff
@@ -236,6 +236,25 @@ in submodules can lead to duplicated `<version>` declarations with different ver
 All version numbers should be easily audited, and so should be in a single file (i.e., the
 *root* `pom.xml`).
 
+
+### THIRD-PARTY LIBRARIES
```
Comment (jgaleotti) [inline docs/for_developers.md]: One of the libraries used by the regex distance code has BSD license, would that be compatible?

File: docs/reused_code.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+* _RegexEcma262.g4_: from [Antlr examples](https://github.com/antlr/grammars-v4/blob/master/ecmascript/ECMAScript.g4).
+  Released under MIT license.
+  
+* _CustomSummaryGeneratingListener.java_ and _CustomMutableTestExecutionSummary.java_:
+  from JUnit 5.
+  Released under Eclipse Public License.  
+  These classes will be removed when this [PR](https://github.com/junit-team/junit5/issues/1947)
+  will be part of the next JUnit release.  
+  
+
```
Comment (jgaleotti) [inline docs/reused_code.md]: Understood, I will add any class added to the code directly here once this PR is merged into master.

File: docs/for_developers.md (hunk 2 above)
Comment (arcuri82) [reply docs/for_developers.md]: As far as I know, only GPL is a problem. other OS licenses are usually fine

File: docs/reused_code.md (hunk 3 above)
Comment (arcuri82) [reply docs/reused_code.md]: if you approve these clarifications, and require no further info, you can just go ahead and merge ;)

**Pull request merged**